lib/config/rules/typescript-react.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  // Disable JS specific rules
+  'react/jsx-filename-extension': 'off',
+  'react/default-props-match-prop-types': 'off',
+
+  // Breaks typescript-eslint-parser
+  'react/jsx-indent': 'off',
```

Comment:
It sucks that the fixers for these rules break the parser, but using the prettier config with the parser option set to typescript gives us the features back.